org.darktable.Darktable.json

```diff
@@ -13,10 +13,8 @@
         "--socket=x11",
         "--filesystem=host",
         "--talk-name=org.freedesktop.secrets",
-        /* Needed for dconf to work */
-        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
-        "--filesystem=xdg-run/gvfs",
-        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
```

Comment:
It was my understanding that this was for when runtime 3.34 is used. darktable still has moved to 3.34 for a few reasons.

Reply:
Nice catch!
I'll revert this change.